app/qml/qml.qrc

```diff
@@ -79,16 +72,24 @@
         <file>editor/inputrangeslider.qml</file>
         <file>editor/inputvaluemap.qml</file>
         <file>editor/inputvaluerelation.qml</file>
+        <file>editor/inputrelationreference.qml</file>
+        <file>editor/AbstractEditor.qml</file>
+        <file>editor/RelationTextDelegate.qml</file>
+        <file>editor/RelationPhotoDelegate.qml</file>
+        <file>editor/RelationPhotoFooterDelegate.qml</file>
         <file>form/FeatureForm.qml</file>
         <file>form/FeatureFormStyling.qml</file>
         <file>form/FeatureToolbar.qml</file>
         <file>form/PreviewPanel.qml</file>
         <file>form/FormWrapper.qml</file>
         <file>form/FeatureFormPage.qml</file>
-        <file>editor/RelationTextDelegate.qml</file>
-        <file>editor/RelationPhotoDelegate.qml</file>
-        <file>editor/RelationPhotoFooterDelegate.qml</file>
-        <file>editor/inputrelationreference.qml</file>
-        <file>editor/AbstractEditor.qml</file>
+        <file>map/MapWrapper.qml</file>
```

Comment:
Since introducing new subfolder (and generally redefining the list), could be good to have it sorted alphabetically.